src/redprl/refiner.fun

```diff
@@ -1477,25 +1477,27 @@ struct
          | (Syn.FST _, Syn.FST _, _) => DProd.FstEq
          | (Syn.SND _, Syn.SND _, _) => DProd.SndEq
          | (Syn.PATH_AP (_, P.VAR _), Syn.PATH_AP (_, P.VAR _), _) => Path.ApEq
-         | (Syn.PATH_AP (_, P.APP _), _, _) => Path.ApComputeConst
+         | (Syn.PATH_AP (_, P.APP _), _, _) => Path.ApConstCompute
          | (_, Syn.PATH_AP (_, P.APP _), _) => Equality.Symmetry
          | _ => raise E.error [E.% "Could not find neutral equality rule for", E.! m, E.% "and", E.! n, E.% "at type", E.! ty]
 
-      fun StepEqEta ty =
-        case Syn.out ty of
-           Syn.DPROD _ => DProd.Eta
-         | Syn.DFUN _ => DFun.Eta
-         | Syn.PATH_TY _ => Path.Eta
-         | _ => raise E.error [E.% "Could not find eta expansion rule for type", E.! ty]
+      fun StepEqNeuVal (m, ty) =
```

Comment:
These changes look good; my only concern is the name, since `ApConstCompute` doesn't turn a neutral into a value, but a neutral into something that may be a redex.

Maybe we should rename `StepEqNeuVal` to `StepComputeNeut`, and `StepEqValVal` back to `StepEqVal`, and `StepEqNeuNeu` to `StepEqNeu`. My thought is that eta and beta principles can be captured under things with the word `...Compute...` in them, and other *structural*/symmetric principles can be captured under `...EqNeu` and `...EqVal`.